CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 ## v1.3.0 [unreleased]
 
+### Release Notes
+
+#### Continuous Query Statistics
+
+When enabled, each time a continuous query is completed, a number of details regarding the execution are written to the `cq_query` measurement of the internal monitor database (`_internal` by default). The tags and fields of interest are
```

Comment:
nit: Stats are only written to `_internal` is `[monitor] store-enabled = true`.  If that is not enabled, they are just available via `show stats` and the debug endpoints.